src/gluonts/model/seq2seq/_mq_dnn_estimator.py

```diff
@@ -111,6 +115,8 @@ def __init__(
         self,
         freq: str,
         prediction_length: int,
+        sampling: bool = True,
+        distr_output: DistributionOutput = StudentTOutput(),
```

Comment:
Same as for the other class

```suggestion
        distr_output: Optional[DistributionOutput] = None,
```

And internally assert that `distr_output is None != quantiles is None` (that's the logical XOR)